src/core/Decal.tsx

```diff
@@ -116,31 +115,33 @@ export const Decal: ForwardRefComponent<DecalProps, THREE.Mesh> = /* @__PURE__ *
 
   React.useLayoutEffect(() => {
     if (helper.current) {
-      applyProps(helper.current as any, state.current)
       // Prevent the helpers from blocking rays
       helper.current.traverse((child) => (child.raycast = () => null))
     }
   }, [debug])
 
-  // <meshStandardMaterial transparent polygonOffset polygonOffsetFactor={-10} {...props} />}
   return (
-    <mesh
-      ref={ref}
-      material-transparent
-      material-polygonOffset
-      material-polygonOffsetFactor={polygonOffsetFactor}
-      material-depthTest={depthTest}
-      material-map={map}
-      {...props}
-    >
-      {children}
-      {debug && (
-        <mesh ref={helper}>
-          <boxGeometry />
-          <meshNormalMaterial wireframe />
-          <axesHelper />
-        </mesh>
-      )}
-    </mesh>
+    <>
+      {/* We add the transforms to another object3d to keep them in the scene graph so editors like triplex.dev can pick them up.  */}
+      <group position={position} rotation={rotation} scale={scale} />
```

Comment:
This lets visual editors that need to have the transform data in the scene graph get access to it. I couldn't add this to the root `mesh` as it breaks the decal geometry expectations.